lib/api/core/models/repositories/repository.js

```diff
@@ -65,13 +66,20 @@ class Repository {
       this.databaseEngine = options.databaseEngine;
     }
     else {
-      this.databaseEngine = options.databaseEngine || this.kuzzle.internalEngine;
+      // @todo refactor storageEngine set (it's dirty to pass it with options.databaseEngine)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)